CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## [Unreleased]
 
+- update jdbc to 0.7.11, update jdbc-util to 1.3.0, due to a change in version for migratus
```

Comment:
should this be under 4.6.30?

Reply:
It wasn't released.  So it will go in the next release.

Reply:
Are we going to do another release PR, or do we plan to release after this is merged?
If it is the latter, we should just move both of these under a `4.7.0` version number.

Reply:
I was assuming we'd just release after this was merged? open to either

Reply:
might be good to do a release before this gets merged and then cut 4.7.0